src/web/server/locales/en/common.json

```diff
@@ -90,7 +90,8 @@
     "title": "We’re sending you a code",
     "heading": "We’re sending you a code",
     "hint": "Select how you would like to receive it",
-    "explanation": "We need this to confirm your identity and to prevent fraud."
+    "explanation": "We need this to confirm your identity and to prevent fraud.",
+    "change": "change"
```

Comment:
Any reason its `change` rather than `Change`?

Reply:
Because it looks better :)
Seriously, that was the UX preference.

Reply:
That's fine, just thought I'd check.